scripts/analyze_training_metrics.py

```diff
@@ -85,12 +85,12 @@ def parse_metric_file(metric_file_path: Path) -> Tuple[datetime, str, Dict[str,
 
     # Get the model, date and version from the file
     # TODO: Might be better storing it in the file
-    file_path_parts = metric_file_path.stem.split("_")
 
-    assert file_path_parts[:5] == ["metric", "project", "bugbug", "train"]
-    model_name = file_path_parts[5]
-    assert file_path_parts[6:8] == ["per", "date"]
-    date_parts = list(map(int, file_path_parts[8:14]))
+    file_path_parts = metric_file_path.stem.split("_")
+    assert file_path_parts[:4] == ["metric", "project", "bugbug", "train"]
+    model_name = file_path_parts[4]
+    assert file_path_parts[5:7] == ["per", "date"]
+    date_parts = list(map(int, file_path_parts[7:13]))
```

Comment:
Could you make a separate PR with this fix? I prefer to have self-contained PRs.

Reply:
Ok done.

Reply:
You'll now need to rebase this PR on top of current master (`git pull upstream master --rebase`).

Reply:
It looks like it didn't go well, https://github.com/mozilla/bugbug/pull/1824/files still shows the changes from the other PR.

Reply:
Ok, got it now I think!